light-zk.js/src/account.ts

```diff
@@ -111,7 +114,13 @@ export class Account {
       this.poseidonEddsaKeypair = Account.getEddsaPrivateKey(
         this.burnerSeed.toString(),
       );
-      this.aesSecret = Account.generateAesSecret(this.burnerSeed.toString());
+      this.aesSecret = Account.generateSecret(
+        this.burnerSeed.toString(),
+        "aes",
+      );
+      this.hashingSecret =
+        hashingSecret ||
```

Comment:
what's the reasoning for passing assigning the passed in hashing secret here?
(feels inconsistent with the lines above)

Reply:
as per the above, I mimic-ed the `aesSecret` scope/meaning/behaviour

Reply:
after code inspection -> removed. ✅